RxCocoa/Traits/SharedSequence/SharedSequence+Operators.swift

```diff
@@ -358,7 +358,7 @@ extension SharedSequence {
     }
 
     /**
-     Concatenates all observable sequences in the given sequence, as long as the previous observable sequence terminated successfully.
```

Comment:
Sequence == Collection, plus the type is `SharedSequence`. I don't think this change is required.